ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -10590,19 +10590,8 @@ static int zend_jit_fetch_obj_read(dasm_State **Dst, const zend_op *opline, cons
 				|	EXT_CALL zend_jit_undefined_op_helper, r0
 				|1:
 			}
```

Comment:
Adding a `GET_ZVAL_PTR FCARG1a, op1_addr` here should do it. I was wrong about the object already being in FCARG1a.

Reply:
Thank you :)

Reply:
Something is still not good with JIT :(

Reply:
Oops, this should have been LOAD_ZVAL_ADDR instead of GET_ZVAL_PTR.

Reply:
Looks like there's still one JIT failure in Zend/tests/bug47109.phpt...

Reply:
Yeah, I saw that :( TBH it's mind bogging that JIT prevents mere mortals like me to do such a small change like this.. Everything seems so cryptic if you are not an expert in assembly + the PHP specific details

Reply:
I've pushed a change to fix this. The problem is that the op1_addr variable is overwritten, while we need the original one.

Reply:
Thank you (again)! I'll new fix the commits, and wait for a green CI :)